src/client/websocket/handlers/INTERACTION_CREATE.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const { Events, InteractionTypes } = require('../../../util/Constants');
+let Structures;
+
+module.exports = (client, { d: data }) => {
+  if (data.type === InteractionTypes.APPLICATION_COMMAND) {
+    if (data.guild_id && !client.guilds.cache.has(data.guild_id)) return;
```

Comment:
This will prevent interactions from guilds without a bot. The types for Interaction#guildId and Interaction#guild already allow the guild to be null if it is not cached.